### PR TITLE
Adds nightlies for citus-enterprise, pgaf and pgaf-enterprise fixes citus nightlies

### PR DESCRIPTION
**Pull request by gurkanindibay**
In this PR  citus-enterprise, pgaf and pgaf-enterprise  nightlies added and citus nightlies applied some changes on citus nightlies 
Docker login is added to fetch docker images with higher throughput.

### CODE REVIEW

File: .github/workflows/build-citus-community-nightlies.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Buildand publish citus nightly package
+name: Build and publish citus nightly packages
```
Comment (onurctirtir) [inline .github/workflows/build-citus-community-nightlies.yml]: very minor, to distinguish from enterprise one:

```suggestion
name: Build and publish citus community nightly packages
```

File: .github/workflows/build-pgaf-community-nightlies.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+name: Build and publish pgaf nightly packages
```
Comment (onurctirtir) [inline .github/workflows/build-pgaf-community-nightlies.yml]: very minor, to distinguish from enterprise one:

```suggestion
name: Build and publish pgaf community nightly packages
```

**Review by onurctirtir — APPROVED** [review]
LGTM.

Given that those files only differ by their MAIN_BRANCH's, can you look to see "if" github actions - yaml structure provides a way to inherit a base yml file then extending it for those four projects ?

Also, I randomly looked into 3-4 jobs and assume that you checked all the jobs in this pr.
After merging, please make sure to verify that we push "expected number of packages" to packagecloud for each project.

File: .github/workflows/build-citus-community-nightlies.yml [hunk 3]
```diff
@@ -29,18 +30,26 @@ jobs:
           - ol/7
           - debian/stretch
           - debian/buster
+          - debian/bullseye
```
Comment (onurctirtir) [inline .github/workflows/build-citus-community-nightlies.yml:33]: nice to see that we immediately started publishing nightlies for this distro!

File: .github/workflows/build-citus-community-nightlies.yml (hunk 1 above)
Comment (gurkanindibay) [reply .github/workflows/build-citus-community-nightlies.yml]: Fixed

File: .github/workflows/build-pgaf-community-nightlies.yml (hunk 2 above)
Comment (gurkanindibay) [reply .github/workflows/build-pgaf-community-nightlies.yml]: Fixed

File: .github/workflows/build-citus-community-nightlies.yml (hunk 3 above)
Comment (gurkanindibay) [reply .github/workflows/build-citus-community-nightlies.yml:33]: Fixed

Comment (onurctirtir) [reply .github/workflows/build-citus-community-nightlies.yml:33]: I guess we will get rid of travis job after merging this pr, right ?

File: .github/workflows/build-pgaf-enterprise-nightlies.yml [hunk 4]
```diff
@@ -0,0 +1,81 @@
+name: Build and publish pgaf enterprise nightly packages
+
```
Comment (onurctirtir) [inline .github/workflows/build-pgaf-enterprise-nightlies.yml:2]: I guess we will get rid of travis job after merging this pr, right ?

Comment (gurkanindibay) [reply .github/workflows/build-pgaf-enterprise-nightlies.yml:2]: Yes I will delete the nightly job description. But it's just a description not file

File: .github/workflows/build-citus-community-nightlies.yml [hunk 5]
```diff
@@ -31,6 +31,7 @@ jobs:
           - debian/stretch
           - debian/buster
           - debian/bullseye
+          - debian/bullseye
```
Comment (onurctirtir) [inline .github/workflows/build-citus-community-nightlies.yml]: why we added one more debian/bullseye in 1b4558a2183851e3d8e13f5184dd1133ddad56c5 ?

File: .github/workflows/build-citus-community-nightlies.yml (hunk 3 above)
Comment (onurctirtir) [reply .github/workflows/build-citus-community-nightlies.yml:33]: I just meant that this is something very nice to see, so there wasn't anything to fix here :)

File: .github/workflows/build-citus-community-nightlies.yml (hunk 5 above)
Comment (gurkanindibay) [reply .github/workflows/build-citus-community-nightlies.yml]: my mistake removed it

File: .github/workflows/build-citus-community-nightlies.yml (hunk 3 above)
Comment (gurkanindibay) [reply .github/workflows/build-citus-community-nightlies.yml:33]: Sorry I misread it

**Pull request merged**